### PR TITLE
2015-06-17-fast-track-to-scala.md

### PR DESCRIPTION
**Pull request by rakeshsingh9901**
This two-day course is designed to give experienced developers proficient in Java, C++, C# or Ruby the know-how to confidently start programming in Scala.

### CODE REVIEW

File: training/2015-06-17-fast-track-to-scala.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+title: Fast Track to Scala
+description: Be an expert in just 2 days (Typesafe Certified Training)
+link-out: http://http://www.springpeople.com/courses/regular/typesafe-certified-fast-track-to-scala-workshop-training-course.php
```
Comment (heathermiller) [inline training/2015-06-17-fast-track-to-scala.md]: URL is wrong

Comment (rakeshsingh9901) [inline training/2015-06-17-fast-track-to-scala.md]: That's why I'm a human. Anyways, I have updated the file.

**Comment by heathermiller** [comment]
Thanks.

**Comment by rakeshsingh9901** [comment]
Thanks to you!

**Comment by heathermiller** [comment]
Your post won't be shown unless you follow the conventions in the README . It's not being parsed as YAML front matter. https://github.com/scala/scala-lang/tree/master/training
Before making pull requests, also please follow instructions and generate the site to make sure there are no errors.

**Comment by heathermiller** [comment]
This is also not even in the correct location. It should be in `training/_posts`.

**Comment by rakeshsingh9901** [comment]
The link of this training is not directing to the link given in the file. Kindly delete this file so that I can create a new one.

**Comment by sjrd** [comment]
@rakeshsingh9901 You can also simply make a PR.

**Pull request merged**